runtime/common/src/impl_self_contained_call.rs

```diff
@@ -72,40 +72,5 @@ macro_rules! impl_self_contained_call {
 				}
 			}
 		}
-
-		fn validate_self_contained_inner(
-			call: &Call,
-			eth_call: &pallet_ethereum::Call<Runtime>,
-			signed_info: &<Call as fp_self_contained::SelfContainedCall>::SignedInfo
-		) -> TransactionValidity {
-			if let pallet_ethereum::Call::transact { ref transaction } = eth_call {
-				// Previously, ethereum transactions were contained in an unsigned
-				// extrinsic, we now use a new form of dedicated extrinsic defined by
-				// frontier, but to keep the same behavior as before, we must perform
-				// the controls that were performed on the unsigned extrinsic.
-				use sp_runtime::traits::SignedExtension as _;
-				let input_len = match transaction {
-					pallet_ethereum::Transaction::Legacy(t) => t.input.len(),
-					pallet_ethereum::Transaction::EIP2930(t) => t.input.len(),
-					pallet_ethereum::Transaction::EIP1559(t) => t.input.len(),
-				};
-				let extra_validation = SignedExtra::validate_unsigned(
```

Comment:
It's a breaking changes to not execute anymore the validate_unsigned checks, are you sure we can do that ?

Reply:
Ok, I dug deeper and I just understood that the only check that validate_unsigned brought us was the one done by CheckWeight, which has been added in frontier, so everything is good.